docs_source/content/resources/wait.port.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "wait.port"
+slug: "wait-port"
+date: "2016-09-30T10:48:08-04:00"
+menu:
+  main:
+    parent: resources
+---
+
+
+
+
+## Example
+
+```hcl
+wait.port "8080" {
+  host         = "localhost"
+  port         = 8080
+  protocol     = "tcp"
```

Comment:
whoops, I'll remove it. I added that before I realized that udp checks are not really feasible.